task_DEV_5/task_DEV_5/GenerateFile.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Text;
+using System.IO;
+
+namespace task_DEV_5
+{
+    class GenerateFile
```

Comment:
Should be noun. Like Generator not Generate.

Reply:
Done.